service/relayer/migrations/20260626120000_slim_relayer_accounting.down.sql

```
@@ -0,0 +1,41 @@
+    "status" = 'pending'::transaction_status;
+
+ALTER TABLE
+    account DROP COLUMN IF EXISTS pending_inner_spend,
+    DROP COLUMN IF EXISTS pending_operation_key,
+ADD
+    COLUMN IF NOT EXISTS pending_transaction_hash varchar(45);
+
+ALTER TABLE
+    account
+ADD
+    CONSTRAINT fk__account__transaction FOREIGN KEY (pending_transaction_hash) REFERENCES "transaction" (transaction_hash);
```

SQLFluff (4.2.2)

[error] ADD CONSTRAINT ... FOREIGN KEY should use NOT VALID to avoid locking the table while validating existing rows.

(PG01)